docs/services/newrelic.md

```diff
@@ -0,0 +1,58 @@
+# NewRelic
+
+## Parameters
+
+* `apiURL` - the api server url, e.g. https://api.newrelic.com
+* `apiKey` - a [NewRelic ApiKey](https://docs.newrelic.com/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2/#api_key)
+
+## Configuration
+
+1. Create a NewRelic [Api Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#user-api-key)
+2. Store apiKey in `argocd-notifications-secret` Secret and configure NewRelic integration in `argocd-notifications-cm` ConfigMap
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <config-map-name>
+data:
+  service.mattermost: |
+    apiURL: <api-url>
+    apiKey: $newrelic-apiKey
+```
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name>
+stringData:
+  newrelic-apiKey: apiKey
+```
+
+3. Copy [Application ID](https://docs.newrelic.com/docs/apis/rest-api-v2/get-started/get-app-other-ids-new-relic-one/#apm)
+4. Create subscription for your NewRelic integration
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  annotations:
+    notifications.argoproj.io/subscribe.<trigger-name>.newrelic: <app-id>
+```
+
+## Templates
```

Comment:
Probably also add description that cover it. ? 

```
commitAuthorTemplate := n.User
	if commitAuthorTemplate == "" {
		commitAuthorTemplate = "{{(call .repo.GetCommitMetadata .app.status.sync.revision).Author}}"
	}
```

You can hardcode user if needed